recipes/rgi/3.2.1/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "rgi" %}
+{% set source_url = "https://github.com/arpcard/rgi_conda_dev.git" %}
+{% set version = "3.2.1" %}
+{% set commit = "77fd4c86cde8b941e19696e85d306e52509e247b" %} 
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: {{ source_url }}
+  git_rev: {{ commit }}
+
+build:
+  skip: True # [py3k]
+  number: 1
+
+# We pin on gcc version to avoid gcc-5 dual ABI issues depending on boost.
```

Comment:
this comment describes pinning gcc, but I don't see any pinning below, in which case you can remove this comment (if it doesn't really need to be pinned)